packages/core/core/src/resolveOptions.js

```diff
@@ -15,6 +15,13 @@ import {NodePackageManager} from '@parcel/package-manager';
 const DEFAULT_CACHE_DIRNAME = '.parcel-cache';
 const LOCK_FILE_NAMES = ['yarn.lock', 'package-lock.json', 'pnpm-lock.yaml'];
 
+// Generate a unique instanceId, will change on every run of parcel
+function generateInstanceId(entries: Array<FilePath>): string {
+  return md5FromString(
```

Comment:
Not entirely sure how we wanted to generate this, so I based this loosely on uuid's with a random value, timestamp and in this case also entrypoints as they probably are unique of itself most of the time